doc/api/stream.md

```diff
@@ -1670,7 +1670,7 @@ failure, this can cause event listener leaks and swallowed errors.
 
 ### `stream.Readable.from(iterable, [options])`
 <!-- YAML
-added: v12.3.0
+added: v10.17.0
```

Comment:
@nodejs/documentation What's the correct metadata here? Can we have multiple `added:` entries (for each release line)? This API never went into a Node.js 11.x release.

Reply:
Yes. We should have multiple added: entries:

```suggestion
added:
  - v12.3.0
  - v10.17.0
```

Reply:
Thanks, Done.